### ISSUE TITLE
keyboard shortcut to close the panel?

### ISSUE THREAD

**Issue by shamaho** [skywind3000/atom-shell-commands#16]
Sorry if this a basic question but... I was wondering if there is a keyboard shortcut to close the atom-shell-command pane...

**Comment by dqsully** [comment]
No, but what shortcut would you like? I will be implementing it in my fork.

**Comment by shamaho** [comment]
Not being a fluent nor experienced atom user yet, I don't have any sort of shortcut recommendation - I'd suggest something in line with the regular or most used shortcut used in Atom to close window panes and such...

Not much helpful I know... :-)

**Comment by dqsully** [comment]
Actually, now that I think of it, `Esc` should probably be the one. I'll put it in later today.

**Comment by skywind3000** [comment]
Sorry, for replying late,
This repository is looking for maintainer now, https://github.com/skywind3000/atom-shell-commands/issues/20
